pkg/cloudprovider/gce/gce.go

```diff
@@ -276,13 +276,18 @@ func (gce *GCECloud) targetPoolURL(name, region string) string {
 
 func waitForOp(op *compute.Operation, getOperation func() (*compute.Operation, error)) error {
 	pollOp := op
+	consecPollFails := 0
 	for pollOp.Status != "DONE" {
 		var err error
-		// TODO: add some backoff here.
```

Comment:
Backoff isn't really useful for polling. But 1 second is too often. How about a 3 second poll, and then just bail on 3 consecutive failures? This _should_ mostly be getting called from reconciler style code, so all we'd really be doing is possibly delaying error reporting.